content/android/get-started.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Get Started"
+date: 2017-07-03T22:20:00-04:00
+draft: false
+mainpage: false
+projectpage: true
+projectslug: "android"
+projectname: "WorldWind Android"
+listdescription: ""
+layout: "project"
+---
+
+## Getting Started
+
+WorldWind Android is an [Android Library](https://developer.android.com/studio/projects/android-library.html) which compiles into an Android Archive (AAR) file. The AAR can be linked and used to display the 3D virtual globe within your application. The WorldWind Android AAR is compatible to API level 19 (Android 4.4 KitKat).
+
+Skip to the [**Download**](#download) section for directions on how to add WorldWind Android to your application. If you need some guidance with setting up the Android Studio IDE, the [**Setup**](#setup) section includes in-depth directions and walks through the creation of a sample project utilizing the WorldWind library.
+
+---
+### <a name="download"></a>Download
```

Comment:
What do you think about using the simpler download/JCenter text I used in Android's README.md? I think it explains enough and conveys the necessary information with much less explanation. Of course it would have to be extended to include OJO.

Again, limit links to those the audience really needs. Over-use of links and formatting makes for a poor reading experience.

Limit formatting to only when it's really needed. Over-use of formatting makes pages an eyesore.